packages/liveblocks-core/src/room.ts

```diff
@@ -832,6 +850,12 @@ type PrivateRoomApi = {
     explicitClose(event: IWebSocketCloseEvent): void;
     rawSend(data: string): void;
   };
+
+  // Used to store metadata related to comments
+  comments: {
+    lastRequestedAt: Date | null;
+    readonly queries: Set<string>;
```

Comment:
@nvie I have added some extra properties related to comments inside the room context and exported as a property of `kInternal`. Wondering if you think it's okay to do so, or if there some things I should be wary about when doing so.

Reply:
Nope! I assume these are for unit testing purposes? Nothing special about them. That said, I already removed a couple not too long ago because I would love to keep this list relatively small. But for now, it's fine to ship private APIs like this 👍

Reply:
These are also internally consumed by `createRoomContext` inside `room.tsx`. The motive is explained here: https://github.com/liveblocks/liveblocks/pull/1450#discussion_r1482163540

I am open to explaining more in detail if it helps.